src/SFA.DAS.ApprenticeCommitments.Web/Services/AuthenticationEvents.cs

```diff
@@ -0,0 +1,48 @@
+﻿using Microsoft.AspNetCore.Authentication.OpenIdConnect;
+using SAF.DAS.ApprenticeCommitments.Web;
+using System;
+using System.Linq;
+using System.Security.Claims;
+using System.Threading.Tasks;
+
+namespace SFA.DAS.ApprenticeCommitments.Web.Services
+{
+    public class AuthenticationEvents : OpenIdConnectEvents
+    {
+        private readonly VerifiedUserService _verifiedUserService;
+
+        public AuthenticationEvents(VerifiedUserService verifiedUserService)
+            => _verifiedUserService = verifiedUserService;
+
+        public override async Task TokenValidated(TokenValidatedContext context)
+        {
+            await base.TokenValidated(context);
+            ConvertRegistrationIdToApprenticeId(context.Principal);
+            await AddUserVerifiedClaim(context.Principal);
+        }
+
+        public void ConvertRegistrationIdToApprenticeId(ClaimsPrincipal principal)
+        {
+            var registrationClaim = principal.Claims.FirstOrDefault(c => c.Type == "registration_id");
+            var apprenticeClaim = principal.ApprenticeIdClaim();
+
+            if (registrationClaim == null) return;
+            if (apprenticeClaim != null) return;
+
+            var apprenticeId = new ClaimsIdentity(new[] { new Claim("apprentice_id", registrationClaim.Value) });
+            principal.AddIdentity(apprenticeId);
+        }
+
```

Comment:
I know this is temporary step, but would it be worthwhile of it to be able to handle both versions of the LoginService. The new one where we do have this 'apprentice_id' claim and the old one where we have the 'registration_id'

Just means when we update the login service, this can be removed later and not at the same time